owncloudTestUtil/build.gradle

```diff
@@ -23,4 +23,5 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion"
     implementation "androidx.lifecycle:lifecycle-livedata-ktx:$lifecycleLiveData"
+    implementation 'androidx.test.espresso:espresso-core:3.3.0'
```

Comment:
Since espresso is only required in the ownCloudApp module, we should try to avoid this dependency anywhere else. It means that if we need to add utility for UI tests, we just need to add it to ownCloudApp since there are not UI tests anywhere else.